libs/utils/executor.py

```diff
@@ -272,13 +272,29 @@ def _setup_group(self, tc, group):
         # Configure each required attribute
         group.set(**tc['cgroups']['conf'][kind][name])
 
+    def _setup_files(self, tc):
+        errors = False
+        if 'files' not in tc:
+            logging.debug('%14s - Configuration not provided', 'Files')
+            return True
+        for name in tc['files']:
+            logging.info('%14s - Write value into file: \'%s\' -> \'%s\'',
```

Comment:
Since the filepath is already quite long and we risk a line overflow
Can we keep this shorter?

Maybe just
```python
 logging.info('%14s - Write: \'%s\' -> \'%s\'',
              'Files', tc['files'][name], name)
```